lib/matplotlib/mpl-data/matplotlibrc

```diff
@@ -701,6 +701,10 @@
                                 # background by default
 #savefig.orientation: portrait  # orientation of saved figure, for PostScript output only
 
+### macosx backend params
+#macosx.window_mode : auto     # How to open new figures (auto, tab, window)
+                               # auto uses system preferences
```

Comment:
Noted in call that maybe `system` instead of `auto` makes more sense here.

Reply:
Updated it to system everywhere now.